draft-dns-zone-digest.xml

```diff
@@ -1101,6 +1119,35 @@ where "|" denotes concatenation.
     </section>
 
     <section anchor="Implementation" title="Implementation Status">
+
+      <t>
+        RFC Editor: Please remove this section before publication.
```

Comment:
I think that we should leave this section in, and replace this line with:
"This section contains information on known implementations at the time of publication. It is expected that this section will become out of date, but we believe that it is useful to keep in the document to provide pointers / examples."

Or similar...